resources/scss/ceres/widgets/Helper/_base-widget.scss

```diff
@@ -51,26 +51,48 @@
     }
 }
 
-#page-header-parent > .widget {
-    max-width: 1200px;
-    width: 100%;
-    background-color: $gray-lightest;
-    
-    &:not(.widget-grid) {
-        max-width: 1170px;
+#page-header-parent {
+
+    > .widget {
+        max-width: 1200px;
+        min-width: 100%;
+        background-color: $gray-lightest;
+        left: 15px;
+        right: 15px;
+
+        &:not(.widget-grid) {
+            max-width: 1170px;
+            min-width: inherit;
+        }
     }
 }
 
-header .widget
-{
-    margin-bottom: 0;
+
+html.scrollbar .widget-fw {
+    padding-left: 7px;
+    padding-right: 7px;
 }
 
-.widget-fw
-{
+#page-header-parent > .widget-fw,
+#page-body > .container-max > .row > .col > .widget-fw,
+.footer > .row > .col > .widget-fw {
```

Comment:
durch diese sehr strikten Selektor-Ketten ist die Wahrscheinlichkeit sehr groß, dass durch Anpassungen in Themes die Utility-Klasse nicht mehr funktioniert. Das sollte nicht der Fall sein. Ich würde hier wie vorher nur die Klasse `.widget-fw` selektieren und über `.widget .widget-fw` die Formatierungen wieder aufheben, um zu verhindern, dass die Klasse auf unteren ebenen verwendet wird.

Reply:
done